### ISSUE TITLE
jalousie or blinds

### ISSUE THREAD

**Issue by hihipto** [Sroose/homebridge-loxone-ws#13]
Hi, thank you for creating this nice and automated integration between homebridge and Loxone.
it is working fine for lights and switches, but my Screens are not being recognized, most of the time I'm using the EIB Blinds. Should I rename them specifically so the ItemFactory.js will recognize them? 
I also want to add the garage door control. eg. open or close gate and request the status of the gate? Is that possible with the current version of Homebridge-loxone-ws?

**Comment by hihipto** [comment]
after some more testing I'm a bit closer to my goal, but not just where I want to be :)
The Jalousie is only recognized by the buildingblock "Blinds" but not by "EIB Blinds" ...
would it be possible to add this to the options like the Dimmer buildingblock?
Will this also display the curtain type of icon vs the blinds icon?
thanks

**Comment by Gusot80** [comment]
I am having the same problem, did you find a solution?

**Comment by hihipto** [comment]
yes Gusot80, I'm using @dgburton fork for curtains and it works fine and on top he added a bunch more components like "Gate" and "Doorbell" 👍 .
However my screens are controlled using the EIB Blinds building block which are not implemented :(
I would hope that they would be added as well (it seems that my programming skills are even worse then I thought ...)

**Comment by Sroose** [comment]
Hi @hihipto,
gate control is now added in release 1.0.5
I don't have any EIB components to test/implement myself.

**Comment by IT-Lueg** [comment]
@hihipto can you tell me how to use the Doorbell component ?
I cant find a way to add a Doorbell from Loxone to Homekit :/

**Comment by hihipto** [comment]
Hi @Sroose ,
Thanks for adding the gate control, it's working fine 👍 
Any news on adding the screens Loxone EIB Blinds buildingblock.
@iLumen I've added the Doorbell as well, but not sure what it adds ...

**Comment by hihipto** [comment]
@Sroose 
ref to the EIB components would it be possible to adjust according:
[https://github.com/Sroose/homebridge-loxone-ws/issues/30](url)
if possible create like a new type eg EIB Blinds?
I would be happy to test them 😄
